src/modules/localStr.js

```diff
@@ -0,0 +1,4 @@
+export const form = document.getElementById('todoform');
+export const todoInput = document.getElementById('newtodo');
+export const todosListEl = document.getElementById('todos-list');
+export const clearBtn = document.getElementById('clear-all-btn');
```

Comment:
- [Optional]  In this file, you have literally just access elements from the HTML document. This could equally have been done in the index.js file. As stated in the requirements, this file was supposed to contain methods related to status updates(completed: true/false)